### PR TITLE
Adds in example of using TourFilters to disable certain tours

### PR DESCRIPTION
**Pull request by warrenbuckley**
Adds in some examples of using `TourFilters` which allows you to filter out specific tours:
* by its alias
* a specific tour JSON filename
* or all JSON tours in an App_Plugin plguin/package

@sofietoft @marcemarc & the rest of the docs gang let me know if you need me to tweak or amend this example.

### CODE REVIEW

File: Extending/Backoffice-Tours/index.md [hunk 1]
```diff
@@ -196,3 +196,37 @@ The image below shows the entire tree highlighted, but requires the user to clic
 ### customProperties
 
 A JSON object that is passed to the scope of a custom step view, so you can use this data in your view with `$scope.model.currentStep.customProperties`.
+
+## How to filter/disable tours being shown
+It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias.
```
Comment (sofietoft) [inline Extending/Backoffice-Tours/index.md:201]: You say it's an example for disabling tours based on alias, right?
In the code example, it's also shown how to disable based on file and based on package - or am I completely off base here? 🤔

Comment (sofietoft) [reply Extending/Backoffice-Tours/index.md:201]: Right! Just noticed your comment 😅 🙈 

So, wouldn't it make sense to mention that in the text as well? That there are 3 different ways to filter out the Tours? 🤔

Comment (warrenbuckley) [inline Extending/Backoffice-Tours/index.md:201]:
```suggestion
It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias., along with showing approaches how you could filter out tours by its JSON filename or by any tour JSON files coming from an App_Plugin plugin/package.
```

File: Extending/Backoffice-Tours/index.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ The image below shows the entire tree highlighted, but requires the user to clic
 A JSON object that is passed to the scope of a custom step view, so you can use this data in your view with `$scope.model.currentStep.customProperties`.
 
 ## How to filter/disable tours being shown
-It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias.
+It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias., along with showing approaches how you could filter out tours by its JSON filename or by any tour JSON files coming from an App_Plugin plugin/package.
```
Comment (sofietoft) [inline Extending/Backoffice-Tours/index.md:201]:
```suggestion
It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection.

Here is an example of disabling all the CMS core tours based on the alias, along with examples on how you could filter out tours by its JSON filename and how to disable tours from specific packages.
```

**Comment by sofietoft** [comment]
Thanks again for adding details on this filtering option @warrenbuckley !
Much appreciated 😄

**Pull request merged**